src/transformState.js

```diff
@@ -6,6 +6,25 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const transform of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
I think it's more logical to call it "action" as you are iterating through actions.
Also, you can make a destructuring to get all the needed props of it (type, extraData, keysToRemove)